src/iinit.cpp

```diff
@@ -433,6 +567,8 @@ void usage( char *prog ) {
     printf( "The client environment file is sought in ~/.irods/irods_environment.json\n" );
     printf( "(unless there is an environment variable indicating a different location),\n" );
     printf( "but the configurations can be set via environment variables as well.\n" );
+    printf( "The client environment file will be updated each time iinit is run in order\n" );
```

Comment:
"The client environment file" -> "The active client environment file"

Reply:
Will do

Reply:
Done